releasenotes/notes/27049.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+
+releaseNotes:
+- |
+  **Added** the ability to use `topology.istio.io/network` and `topology.istio.io/gatewayPort` on the gateway's Kubernetes Service rather than [meshNetworks](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks) to mark a service as a gateway. 
```

Comment:
I'm not sure how to document that gatewayPort is 'experimental'  - not even sure we should document it. I would remove it from release notes, keep it as 'experimental' until we have a clear understanding how BTS config will work ( most likely will be a URL ).

Reply:
Removed from release notes.